pandas/tests/io/parser/test_common.py

```diff
@@ -1062,14 +1062,17 @@ def test_skip_initial_space(all_parsers):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.filterwarnings("ignore:Lenght of header:pandas.errors.ParserWarning")
```

Comment:
```suggestion
@pytest.mark.filterwarnings("ignore:Length of header:pandas.errors.ParserWarning")
```

Reply:
Something went wrong with strg+z here. This should not be in there, removed it.